### PR TITLE
chore(oma-datagov): 

### PR DESCRIPTION
**Pull request by JimHagan**
## Give us some context

* Added explanation on APM events limits to optimization section
* Discussion with Nic Benders explained how this is used in New Relic's own monitoring practices use this to reduce volume.

### CODE REVIEW

File: src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-optimizing.mdx [hunk 1]
```diff
@@ -250,6 +250,35 @@ Common database query setting changes include:
 
 More details can be found [here](docs/apm/transactions/transaction-traces/transaction-traces-database-queries-page/#settings).
 
+### Setting Event Limits
+
+Our APM agents and mobile agents have limits on how many events can be reported per harvest cycle. This is necessary because if there were no limit, a very large number of events being sent could have performance impacts on your application or on New Relic. When the limit is reached, the agents begin sampling events. Different agents have different limits, but the goal is to give a representative sample from across the harvest cycle.
+Events that are capped and subject to sampling include:
+
+- Transaction
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-optimizing.mdx:258]: @JimHagan is there a reason you didn't sort this list alphabetically?

**Review by paperclypse — APPROVED** [review]
I've made some style updates, if you'd like to review them, @JimHagan. Otherwise, I think it's ready to go.

File: src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-optimizing.mdx (hunk 1 above)
Comment (JimHagan) [reply src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-optimizing.mdx]: @paperclypse That list is verbatim from another section of docs.  I would say they probably sorted them to have **_Transaction_** first as it's our most common event type.

I don't have a preference.

Comment (paperclypse) [reply src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-optimizing.mdx]: If you don't have a preference, I'll make it alphabetical. Once you have more than 5 items in a list, it makes it much easier to scan :)

**Pull request merged**